ocs_ci/framework/exceptions.py

```diff
@@ -1,10 +1,30 @@
+from ocs_ci.ocs.constants import (
+    CLUSTER_NAME_MIN_CHARACTERS,
+    CLUSTER_NAME_MAX_CHARACTERS
+)
+
 
 class ClusterPathNotProvidedError(Exception):
 
     def __str__(self):
         return "Please provide a --cluster-path that is empty or non-existant."
 
 
+class ClusterNameLengthError(Exception):
+    def __init__(
+        self, name, min=CLUSTER_NAME_MIN_CHARACTERS, max=CLUSTER_NAME_MAX_CHARACTERS
```

Comment:
Apologies for catching this so late, but I think we might want to use names other than min/max for these arguments as they shadow the `min` and `max` builtin functions. I don't want to hold up this PR so I'm going to approve anyways, but perhaps we can change them to `minimum`/`maximum`, `min_length`/`max_length`, or something like this in a follow up PR.

Reply:
good point!, luckily within this class scope so bit safe.

Reply:
Good catch! I'll submit another PR to fix it.

Reply:
See #1156 for a fix